test/module/irohad/simulator/simulator_test.cpp

```diff
@@ -75,12 +86,16 @@ TEST_F(SimulatorTest, ValidWhenPreviousBlock) {
   auto proposal = model::Proposal(txs);
   proposal.height = 2;
 
-  model::Block block;
-  block.height = proposal.height - 1;
+  shared_model::proto::Block block = makeBlock(proposal.height - 1);
+  auto blocks_observable = rxcpp::observable<>::create<
+      std::shared_ptr<::shared_model::interface::Block>>([&block](auto s) {
+
+    s.on_next(std::shared_ptr<::shared_model::interface::Block>(block.copy()));
```

Comment:
Same as before, `just(block).map(wBlock(block.copy))`. Shortcut lambda or method can be defined, because it is used at least two times in this file.